components/mediation-admin/org.wso2.carbon.rest.api/src/main/java/org/wso2/carbon/rest/api/service/RestApiAdmin.java

```diff
@@ -709,6 +711,23 @@ private APIData convertApiToAPIData(API api) {
             apiData.setTracingEnable(false);
         }
 
+        Handler[] handlers = api.getHandlers();
+        HandlerData[] handlerDatas = new HandlerData[handlers.length];
+        for (int i = 0; i < handlers.length; i++) {
+            HandlerData data = new HandlerData();
+            String handler = handlers[i].toString();
+            data.setHandler(handler.substring(0, handler.lastIndexOf("@")));
+            Map<String, String> propertiesMap = handlers[i].getProperties();
+            ArrayList<String> properties = new ArrayList<>();
+            for (Map.Entry<String, String> entry : propertiesMap.entrySet())
+            {
+                properties.add(entry.getKey() + "::::" + entry.getValue());

```

Comment:
let's define "::::" as constant since used in multiple places
also add come comment at constant definition and here at logic as well